backend/app/services/db_service.py

```
@@ -0,0 +1,326 @@
+"""
+データベース操作サービス
+"""
+import logging
+from typing import List, Dict, Any, Optional
+from app.core.db import Database
+from app.core.exceptions import DatabaseError
+from app.core.error_codes import (
+    DB_CONNECTION_ERROR,
```

Ruff (0.11.9)

`app.core.error_codes.DB_CONNECTION_ERROR` imported but unused

Remove unused import: `app.core.error_codes.DB_CONNECTION_ERROR`

(F401)